packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -39,9 +42,56 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       activeAppearance,
       image: imageProps,
       badge: badgeProps,
+      initials: _initials,
     },
     state: {
       ...state,
     },
   };
 };
+
+/**
+ * A function which takes a name and returns initials
+ *
+ * @param name
+ * @returns initials. First letter of the word if only one word was provided.
+ * First letters of the first and last words if more words were provided.
+ * Words in braces, titles, special characters, parantheses and dashes should be ignored.
+ */
+export const getInitials = (name: string): string => {
+  if (!name && !validateAlphabeticalCharacters(name)) {
+    return '';
+  }
+  // prettier-ignore
```

Comment:
>// prettier-ignore [](http://example.com/codeflow?start=2&length=18)

Why are we having prettier ignore this?

Reply:
On my machine, prettier would start removing some of the slashes in the regex string when I saved the file. That kept messing up the regex. Note: We should add a comment explaining why we're ignoring + one comment for each line to ignore (I think like 67 won't be ignored by prettier for example)

Reply:
Yeah, the same thing on mine. I'll add comment

Reply:
After I fixed regexps (used new RegExp), prettier doesn't remove anything